model_info/viwiki.articletopic.md

```diff
@@ -554,408 +554,408 @@ Model Information:
+		Culture.Visual arts.Visual arts*             0.972
+		Geography.Geographical                       0.968
+		Geography.Regions.Africa.Africa*             0.934
+		Geography.Regions.Africa.Central Africa      0.953
+		Geography.Regions.Africa.Eastern Africa      0.907
+		Geography.Regions.Africa.Northern Africa     0.939
+		Geography.Regions.Africa.Southern Africa     0.951
+		Geography.Regions.Africa.Western Africa      0.965
+		Geography.Regions.Americas.Central America   0.912
+		Geography.Regions.Americas.North America     0.926
+		Geography.Regions.Americas.South America     0.936
+		Geography.Regions.Asia.Asia*                 0.915
+		Geography.Regions.Asia.Central Asia          0.957
+		Geography.Regions.Asia.East Asia             0.946
+		Geography.Regions.Asia.North Asia            0.968
+		Geography.Regions.Asia.South Asia            0.932
+		Geography.Regions.Asia.Southeast Asia        0.908
+		Geography.Regions.Asia.West Asia             0.95
+		Geography.Regions.Europe.Eastern Europe      0.951
+		Geography.Regions.Europe.Europe*             0.926
+		Geography.Regions.Europe.Northern Europe     0.921
+		Geography.Regions.Europe.Southern Europe     0.934
+		Geography.Regions.Europe.Western Europe      0.94
+		Geography.Regions.Oceania                    0.923
+		History and Society.Business and economics   0.97
+		History and Society.Education                0.972
+		History and Society.History                  0.948
+		History and Society.Military and warfare     0.975
+		History and Society.Politics and government  0.922
+		History and Society.Society                  0.899
+		History and Society.Transportation           0.986
+		STEM.Biology                                 0.986
 		STEM.Chemistry                               0.988
-		STEM.Computing                               0.987
-		STEM.Earth and environment                   0.98
-		STEM.Engineering                             0.986
-		STEM.Libraries & Information                 0.974
-		STEM.Mathematics                             0.989
-		STEM.Medicine & Health                       0.981
-		STEM.Physics                                 0.988
-		STEM.STEM*                                   0.983
-		STEM.Space                                   0.994
-		STEM.Technology                              0.981
+		STEM.Computing                               0.986
+		STEM.Earth and environment                   0.974
+		STEM.Engineering                             0.983
+		STEM.Libraries & Information                 0.964
+		STEM.Mathematics                             0.984
+		STEM.Medicine & Health                       0.976
+		STEM.Physics                                 0.987
+		STEM.STEM*                                   0.982
+		STEM.Space                                   0.99
+		STEM.Technology                              0.977
 		-------------------------------------------  -----
-	pr_auc (micro=0.826, macro=0.684):
```

Comment:
@halfak Macro is higher, but micro is lower at cutoff 20000.

The values at cutoff 10000 were:

pr_auc (micro=0.711, macro=0.533)